helm/README.md

```diff
@@ -21,9 +21,9 @@ The `fiftyone-teams-app`, `fiftyone-teams-api`, and `fiftyone-app` images are av
 
 ## Initial Installation vs. Upgrades
 
-`FIFTYONE_DATABASE_ADMIN` is set to `false` by default for FiftyOne Teams version 1.2.1 upgrades and installations. This is because FiftyOne Teams version 1.2.1 is backwards compatible with FiftyOne Teams database schema 0.19 (Teams Version 1.1).
+`FIFTYONE_DATABASE_ADMIN` is set to `false` by default for FiftyOne Teams version 1.3.0 upgrades. This is because FiftyOne Teams version 1.3.0 is backwards compatible with FiftyOne Teams database schema 0.20 (Teams Version 1.2).
```

Comment:
No idea if helm is different or this bump intentional, but should this be db v0.19/Teams v1.1 like mentioned elsewhere?